resources/icons/16px/antenna.svg

```diff
@@ -0,0 +1,3 @@
+<path fill-rule="evenodd" clip-rule="evenodd" d="M2.67261 10.7577C2.68007 10.7722 2.68724 10.7867 2.69412 10.8013C2.70553 10.8256 2.71613 10.8501 2.72595 10.8749C2.74125 10.9135 2.75461 10.9528 2.76602 10.9924C2.89304 11.4337 2.77766 11.9239 2.40414 12.2045C1.96261 12.5363 1.32909 12.4501 1.05497 11.9707C0.492513 10.9869 0.143532 9.89133 0.035771 8.75563C-0.103594 7.28685 0.165927 5.80813 0.814535 4.48297C1.46314 3.15781 2.46557 2.03783 3.711 1.24684C4.95642 0.455851 6.39632 0.0246739 7.87151 0.000977833C9.3467 -0.0227182 10.7997 0.36199 12.0699 1.11257C13.3401 1.86315 14.378 2.95036 15.0688 4.25401C15.7596 5.55765 16.0765 7.02695 15.9844 8.49945C15.9132 9.63803 15.5995 10.7442 15.069 11.7456C14.8104 12.2336 14.18 12.34 13.728 12.0226C13.3457 11.7541 13.2146 11.2679 13.3274 10.8228C13.3375 10.7828 13.3496 10.7431 13.3637 10.704C13.3727 10.6789 13.3825 10.6541 13.3931 10.6295C13.3995 10.6146 13.4062 10.5999 13.4132 10.5852C13.743 9.89464 13.939 9.14444 13.9872 8.3745C14.0563 7.27033 13.8186 6.16857 13.3006 5.19102C12.7826 4.21347 12.0043 3.39821 11.0519 2.83538C10.0994 2.27255 9.00984 1.98407 7.90365 2.00184C6.79747 2.01961 5.71774 2.34293 4.78385 2.93606C3.84995 3.52919 3.09827 4.36903 2.61191 5.36271C2.12554 6.35639 1.92344 7.46522 2.02794 8.5666C2.10082 9.3346 2.32079 10.0781 2.67261 10.7577ZM5.13673 8.89614C5.15801 8.96412 5.17531 9.03375 5.18834 9.10418C5.20037 9.16913 5.20877 9.23476 5.21333 9.3004C5.24215 9.71546 5.11738 10.131 4.78343 10.3779C4.33938 10.7062 3.70261 10.6148 3.46908 10.1144C3.3751 9.91297 3.29465 9.70554 3.22828 9.49349L3.22499 9.48293C3.11942 9.14303 3.04998 8.79132 3.01884 8.43354C2.93887 7.5149 3.1145 6.59204 3.52631 5.76699C3.93811 4.94194 4.57006 4.24684 5.35227 3.75854C6.13448 3.27024 7.03648 3.00776 7.95856 3.00012C8.88064 2.99248 9.78688 3.23997 10.5771 3.71524C11.3673 4.19051 12.0106 4.87504 12.4361 5.69315C12.8615 6.51126 13.0524 7.43108 12.9877 8.35092C12.9625 8.70916 12.8989 9.06198 12.7989 9.40358L12.7958 9.41419C12.733 9.62732 12.656 9.83605 12.5653 10.039C12.3401 10.5432 11.705 10.6451 11.2555 10.3243C10.9175 10.0829 10.7859 9.66952 10.8078 9.25404C10.8113 9.18833 10.8186 9.12257 10.8296 9.05743C10.8414 8.98679 10.8576 8.91688 10.8777 8.84856C10.9388 8.6416 10.9776 8.42774 10.9928 8.21055C11.0317 7.6586 10.9171 7.10666 10.6619 6.61576C10.4066 6.12485 10.0205 5.7141 9.54637 5.42891C9.07221 5.14373 8.52843 4.99522 7.97514 4.9998C7.42184 5.00439 6.88059 5.16189 6.41123 5.4549C5.94187 5.7479 5.56267 6.16499 5.31557 6.66006C5.06846 7.15513 4.96307 7.70889 5.01106 8.26012C5.02994 8.47703 5.07228 8.69022 5.13673 8.89614ZM7.99999 6.99989C8.55228 6.99989 8.99999 7.44761 8.99999 7.99989V11.5857L11.7071 14.2928C12.0976 14.6833 12.0976 15.3165 11.7071 15.707C11.3166 16.0975 10.6834 16.0975 10.2929 15.707L7.99999 13.4141L5.7071 15.707C5.31658 16.0975 4.68341 16.0975 4.29289 15.707C3.90236 15.3165 3.90236 14.6833 4.29289 14.2928L6.99999 11.5857V7.99989C6.99999 7.44761 7.44771 6.99989 7.99999 6.99989Z" fill="black"/>
```

Comment:
nit: `fill="black"` (at the end) isn't needed (same applies to the other files)

Reply:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M2.67261 10.7577C2.68007 10.7722 2.68724 10.7867 2.69412 10.8013C2.70553 10.8256 2.71613 10.8501 2.72595 10.8749C2.74125 10.9135 2.75461 10.9528 2.76602 10.9924C2.89304 11.4337 2.77766 11.9239 2.40414 12.2045C1.96261 12.5363 1.32909 12.4501 1.05497 11.9707C0.492513 10.9869 0.143532 9.89133 0.035771 8.75563C-0.103594 7.28685 0.165927 5.80813 0.814535 4.48297C1.46314 3.15781 2.46557 2.03783 3.711 1.24684C4.95642 0.455851 6.39632 0.0246739 7.87151 0.000977833C9.3467 -0.0227182 10.7997 0.36199 12.0699 1.11257C13.3401 1.86315 14.378 2.95036 15.0688 4.25401C15.7596 5.55765 16.0765 7.02695 15.9844 8.49945C15.9132 9.63803 15.5995 10.7442 15.069 11.7456C14.8104 12.2336 14.18 12.34 13.728 12.0226C13.3457 11.7541 13.2146 11.2679 13.3274 10.8228C13.3375 10.7828 13.3496 10.7431 13.3637 10.704C13.3727 10.6789 13.3825 10.6541 13.3931 10.6295C13.3995 10.6146 13.4062 10.5999 13.4132 10.5852C13.743 9.89464 13.939 9.14444 13.9872 8.3745C14.0563 7.27033 13.8186 6.16857 13.3006 5.19102C12.7826 4.21347 12.0043 3.39821 11.0519 2.83538C10.0994 2.27255 9.00984 1.98407 7.90365 2.00184C6.79747 2.01961 5.71774 2.34293 4.78385 2.93606C3.84995 3.52919 3.09827 4.36903 2.61191 5.36271C2.12554 6.35639 1.92344 7.46522 2.02794 8.5666C2.10082 9.3346 2.32079 10.0781 2.67261 10.7577ZM5.13673 8.89614C5.15801 8.96412 5.17531 9.03375 5.18834 9.10418C5.20037 9.16913 5.20877 9.23476 5.21333 9.3004C5.24215 9.71546 5.11738 10.131 4.78343 10.3779C4.33938 10.7062 3.70261 10.6148 3.46908 10.1144C3.3751 9.91297 3.29465 9.70554 3.22828 9.49349L3.22499 9.48293C3.11942 9.14303 3.04998 8.79132 3.01884 8.43354C2.93887 7.5149 3.1145 6.59204 3.52631 5.76699C3.93811 4.94194 4.57006 4.24684 5.35227 3.75854C6.13448 3.27024 7.03648 3.00776 7.95856 3.00012C8.88064 2.99248 9.78688 3.23997 10.5771 3.71524C11.3673 4.19051 12.0106 4.87504 12.4361 5.69315C12.8615 6.51126 13.0524 7.43108 12.9877 8.35092C12.9625 8.70916 12.8989 9.06198 12.7989 9.40358L12.7958 9.41419C12.733 9.62732 12.656 9.83605 12.5653 10.039C12.3401 10.5432 11.705 10.6451 11.2555 10.3243C10.9175 10.0829 10.7859 9.66952 10.8078 9.25404C10.8113 9.18833 10.8186 9.12257 10.8296 9.05743C10.8414 8.98679 10.8576 8.91688 10.8777 8.84856C10.9388 8.6416 10.9776 8.42774 10.9928 8.21055C11.0317 7.6586 10.9171 7.10666 10.6619 6.61576C10.4066 6.12485 10.0205 5.7141 9.54637 5.42891C9.07221 5.14373 8.52843 4.99522 7.97514 4.9998C7.42184 5.00439 6.88059 5.16189 6.41123 5.4549C5.94187 5.7479 5.56267 6.16499 5.31557 6.66006C5.06846 7.15513 4.96307 7.70889 5.01106 8.26012C5.02994 8.47703 5.07228 8.69022 5.13673 8.89614ZM7.99999 6.99989C8.55228 6.99989 8.99999 7.44761 8.99999 7.99989V11.5857L11.7071 14.2928C12.0976 14.6833 12.0976 15.3165 11.7071 15.707C11.3166 16.0975 10.6834 16.0975 10.2929 15.707L7.99999 13.4141L5.7071 15.707C5.31658 16.0975 4.68341 16.0975 4.29289 15.707C3.90236 15.3165 3.90236 14.6833 4.29289 14.2928L6.99999 11.5857V7.99989C6.99999 7.44761 7.44771 6.99989 7.99999 6.99989Z"/>
```